app/src/main/res/values-de/strings.xml

```diff
@@ -153,16 +153,16 @@
     <string name="helper_request_detail_message_cancelled">"Anfrage wurde gelöscht"</string>
     <string name="helper_request_detail_message_error_on_cancellation">"Anfrage konnte nicht gelöscht werden"</string>
     <string name="select_text_dialog_text_hint_default">"Eingeben"</string>
-    <string name="user_input_details_placeholder_firstname">"Vorname"</string>
-    <string name="user_input_details_placeholder_lastname">"Nachname"</string>
+    <string name="user_input_details_placeholder_first_name">"Vorname"</string>
```

Comment:
this rename will probably create a duplicate in POEditor because it is only adding strings but not removing them. Means as soon as this PR is merged, new strings will be created and old ones will be automatically commited by POEditor into develop :(

Reply:
We agreed upon it being ok for this specific time, as it is planned to compare all strings with the iOS strings and correct any error.